cmd/integration_test/build/buildspecs/vsphere-test-eks-a-cli.yml

```diff
@@ -14,6 +14,12 @@ env:
     T_VSPHERE_TEMPLATE_UBUNTU_1_25: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-25"
     T_VSPHERE_TEMPLATE_UBUNTU_1_26: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-26"
     T_VSPHERE_TEMPLATE_UBUNTU_1_27: "/SDDC-Datacenter/vm/Templates/ubuntu-kube-v1-27"
+    T_VSPHERE_TEMPLATE_UBUNTU_2204_1_22: "/SDDC-Datacenter/vm/Templates/ubuntu-2204-kube-v1-22"
```

Comment:
Same as above 1.22 isn't supported anymore. We don't need to include this

Reply:
Done!